packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -89,9 +89,14 @@ export function useComboBox<T>(props: AriaComboBoxOptions<T>, state: ComboBoxSta
       isDisabled: isDisabled || isReadOnly
     },
     state,
+    // useMenuTrigger shouldn't have an optional ref, but because we handle the keyboard
+    // we can omit this. Maybe useComboboxMenuTrigger should be created?
+    // @ts-ignore
     buttonRef
   );
 
+  // Do we throw an error if menuProps.id doesn't exist? that would mean a change in useMenuTrigger which
+  // currently always returns an id.
```

Comment:
I think it's an old comment to myself because I was trying to type the returned value of menuProps and was struggling with it for some reason I cannot recall anymore. Anyhow, I think it's irrelevant now.